api/verify_test.go

```diff
@@ -96,6 +96,75 @@ func (ts *VerifyTestSuite) TestVerify_PasswordRecovery() {
 	assert.True(ts.T(), u.IsConfirmed())
 }
 
+func (ts *VerifyTestSuite) TestVerify_SecureEmailChange() {
```

Comment:
qq: is the convention of `<test_suite>_fnname` a convention within our codebase or a general Go Testing convention? Seems a little odd to mix CamelCase and snake case.

Reply:
yeah good point its not a convention, was just following the work done previously but lets standardise it to CamelCase throughout